editoast/src/views/timetable/stdcm.rs

```diff
@@ -91,7 +93,7 @@ pub struct STDCMRequestPayload {
     /// Specifies how long the total run time can be in milliseconds
     /// Deprecated, first step data should be used instead
     maximum_run_time: Option<u64>,
-    /// Train categories for speed limits
+    /// Train categories for speed limit
```

Comment:
Apparently several speed limits can apply to a train category so it seems we should keep the `limits` plural. However it might need to be renamed to `train category` singular if a train can only have at most one category ? That's not linked to my PR: I am reverting my change and adding a TODO if that's fine with you.

Reply:
A "code composition" is modeled by a speed limit tag. A train has only one speed limit tag, but a speed limit tag can be missing and in this case a fallback system is used.